omrtrace/omrtracewrappers.cpp

```diff
@@ -56,7 +56,7 @@ twThreadSelf(void)
 omr_error_t
 twE2A(char *str)
 {
-#if defined(J9ZOS390)
+#if defined(J9ZOS390) && !defined(OMR_EBCDIC)
```

Comment:
I'm not sure I see the value of decoupling here. As far as I could tell from the existing uses we are trying to accomplish the same thing, i.e. assume native encoding and do not do any runtime ASCII/EBCDIC conversions which is exactly what we're trying to accomplish in the runtime. If files are properlly filetagged with `chtag` (which they are automatically if checked out via Git for z/OS) we do not need any runtime conversions between ASCII/EBCDIC and we should use the platform native APIs.

Reply:
the logic was that enabling native ebcdic handling is a safe enough flag to enable in production (since any failures would show up at build time). Where as enabling it in the runtime is more tricky and requires more thorough testing to ensure there are no bugs that will only show up on users machines

Reply:
Fixed as part of 5682d6e and 91f1ee5